src/js/utils/utils.js

```diff
@@ -312,6 +309,49 @@ const utils = {
 
         audio.preload = 'metadata';
         audio.src = path;
+    },
+
+    fetchCover(trackPath, callback) {
+
+        if(!trackPath) {
+            callback(null);
+            return;
+        }
+
+        const stream = fs.createReadStream(trackPath);
+
+        mmd(stream, (err, data) => {
+
+            if(err) console.warn(err);
+            else {
+                if(data.picture[0]) { // If cover in id3
+
+                    const cover = utils.parseBase64(data.picture[0].format, data.picture[0].data.toString('base64'));
+
+                    callback(cover);
+                    return;
+                }
+
+                // scan folder for any cover image
+                const folder = path.dirname(trackPath);
+
+                const pattern = path.join(folder, '*');
+
+                globby(pattern, { nodir: true, follow: false }).then((matches) => {
+
+                    const cover = matches.find((elem) => {
+
+                        const parsedPath = path.parse(elem);
+
+                        return ['album', 'albumart', 'folder', 'cover'].includes(parsedPath.name.toLowerCase())
+                            && ['.png', '.jpg', '.bmp', '.gif'].includes(parsedPath.ext.toLowerCase()) ;
+                    });
+
+                    callback(cover);
```

Comment:
It returns undefined, then https://github.com/KeitIG/museeks/pull/195/files#diff-6faf57b394e7c8c1a51f2c5c9f4cd5f5R421 handles it